recipes/nss/all/conanfile.py

```diff
@@ -17,182 +18,182 @@ class NSSConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     description = "Network Security Services"
     topics = ("network", "security", "crypto", "ssl")
-    settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False]}
```

Comment:
Removal of the shared option breaks the Qt recipe with shared=True.

Reply:
@Rodarin break how? Could you please give a build log or some consistent detail to understand? For Conan 2.x build in CCI, the CI service uses `-o */*:shared=True/False`, so in case using static library, it will link shared x static. Not the best scenario, but should work as well.

Reply:
The break is actually caused by a slightly different issue, so I will place a comment at that spot. Sorry for the confusion.